entity-framework/core/modeling/sequences.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Sequences - EF Core
+author: roji
+ms.date: 12/18/2019
+ms.assetid: 94f81a92-3c72-4e14-912a-f99310374e42
+uid: core/modeling/sequences
+---
+# Sequences
+
+> [!NOTE]  
+> Sequences are a feature typically supported only by relational databases. If you're using a non-relational database such as Cosmos, check you database documentation on generating unique values.
+
+A sequence generates unique, sequential numeric values in the database. Sequences are not associated with a specific table, and multiple tables can be set up to draw values from the same sequence.
+
+## Basic usage
+
+You can use the Fluent API to create a sequence in the model, and then configure use it to generate values for properties:
```

Comment:
Should we continue saying "Fluent API"? "ModelBuilder API" might be easier to understand.

Reply:
Yeah, I actually stopped saying "Fluent API", opting for the simpler "you can configure" or "you can set up". Have done the same here, thanks.